autotx/tests/agents/token/test_swap.py

```diff
@@ -108,9 +119,11 @@ def test_auto_tx_swap_complex_2(configuration, auto_tx): # This one is complex b
 
     prompt = "Buy 1000 USDC with ETH, then sell the USDC to buy 0.001 WBTC"
     usdc_balance = manager.balance_of(usdc_address)
-    wbtc_balance = manager.balance_of(wbtc_address)
 
     auto_tx.run(prompt, non_interactive=True)
 
+    wbtc_balance = manager.balance_of(wbtc_address)
+    expected_amount = 0.001
+    expected_amount_plus_slippage = expected_amount * PLUS_DIFFERENCE_PERCENTAGE
+    assert expected_amount <= wbtc_balance and wbtc_balance <= expected_amount + expected_amount_plus_slippage
```

Comment:
Would be cleaner to do:
`assert expected_amount <= wbtc_balance <= expected_amount * DIFFERENCE_PERCENTAGE`
This is using DIFFERENCE_PERCENTAGE (>1) instead of PLUS_DIFFERENCE_PERCENTAGE 
as well as `a < b < c` syntax

Also, I'd rename `expected_amount` to `expected__wbtc_amount` or `expected_wbtc`

Reply:
when we do `expected_amount * PLUS_DIFFERENCE_PERCENTAGE` we are getting the amount of that percentage (example: `0.001 * 0.007 = 0,000007`) that's why are adding it to the expected amount

unless i am misunderstanding something, i think that doing `wbtc_balance <= expected_amount * DIFFERENCE_PERCENTAGE` doesn't make sense?

Reply:
i updated it to use the syntax `a < b < c`

Reply:
I'm saying `0.001 + 0.001 * 0.007 = 0.001 * 1.007`
So making `PLUS_DIFFERENCE_PERCENTAGE` equal `1.007` makes it more readable (and then the PLUS_ prefix stops making sense so that's why the rename to `DIFFERENCE_PERCENTAGE`

Reply:
thanks for clarifying, just pushed the change